src/App.js

```diff
@@ -1,24 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import HomePage from './components/Books';
+import CategoriesPage from './components/Categories';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Routes>
+      <Route path="/" element={<HomePage />} />
+      <Route path="/categories" element={<CategoriesPage />} />
+    </Routes>
```

Comment:
- [ ] Please add Navbar before the `Routes` tag and remove Navbar inside the `Categories.js` file and `Book.js` file because if you use the navbar in the `App.js` file This will apply to every components :slightly_smiling_face:

Reply:
- Refactored `Categories.js` file and `Book.js file` and added `Navbar` functionality into `App.js` file for DRY best practice.